sbt_common/internal_model/src/main/scala/uk/ac/wellcome/models/reindexer/ReindexableRecord.scala

```diff
@@ -7,5 +9,6 @@ package uk.ac.wellcome.platform.reindex_request_creator.models
   */
 case class ReindexableRecord(
   id: String,
+  version: Int,
   reindexVersion: Int
-)
+) extends Id
```

Comment:
possibly a merge conflict, but Id has been expunged from master, so you'll want to drop this